code/style.css

```diff
@@ -1,5 +1,333 @@
+  padding: 10px 20px;
+  margin-bottom: 20px;
+  border: 1px solid #3e3e3e;
+  border-radius: 4px;
+  background-color: #2c2c2c;
+  color: white;
+}
+
+
+input::placeholder {
+  color: #b3b3b3;
+}
+
+
+.radio-group {
+  text-align: left;
+  margin-bottom: 20px;
+}
+
+
+.radio-group p {
+  font-size: 16px;
+  color: #fff;
+  margin-bottom: 15px;
+}
+
+
+.radio-options {
+  display: flex;
+  justify-content: space-around;
+  max-width: 600px;
+  margin: 0 auto;
+}
+
+
+.radio-options label {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 14px;
+  color: #fff;
+  width: 120px;
+  gap: 10px;
+  line-height: 1.5;  
+  flex-direction: row-reverse;
+}
+
+
+.radio-options input[type="radio"] {
+  transform: translateY(-1px); 
+}
+
+
+.checkbox-group {
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  margin-top: 20px;
+  font-size: 1px;
+  color: #fff;
+  padding: 10px;
+}
+
+
+.checkbox-group label {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row-reverse;
+}
+
+
+.checkbox-group input {
+  margin-left: 10px;
+  transform: translateY(-2px); /* Fine-tune the alignment of the checkbox */
+}
+
+
+button[type="submit"] {
+  width: 100%;
+  padding: 10px;
+  margin-bottom: 20px;
+  background-color: #1DB954;
+  border: none;
+  border-radius: 50px;
+  color: white;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+
+button[type="submit"]:hover {
+  background-color: #1aa34a;
+}
+
+
+.divider {
+  position: relative;
+  margin: 20px 0;
+  font-size: 12px;
+  color: #b3b3b3;
+}
+
+
+.divider:before,
+.divider:after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  width: 40%;
+  height: 1px;
+  background-color: #3e3e3e;
+}
+
+
+.divider:before {
+  left: 0;
+}
+
+
+.divider:after {
+  right: 0;
+}
+
+
+.social-signup {
+  margin-bottom: 20px;
+}
+
+
+.social-button {
+  display: block;
+  width: 100%;
+  padding: 10px;
+  margin-bottom: 10px;
+  border-radius: 50px;
+  border: none;
+  cursor: pointer;
+  font-size: 14px;
+}
+
+
+.google {
+  background-color: #4285F4;
+  color: white;
+}
+
+
+.facebook {
+  background-color: #4267B2;
+  color: white;
+}
+
+
+.apple {
+  background-color: #000000;
+  color: white;
+}
+
+
+.login-link {
+  font-size: 12px;
+  color: #b3b3b3;
+}
+
+
+.login-link a {
+  color: #1DB954;
+  text-decoration: none;
+}
+
+
+.login-link a:hover {
+  text-decoration: underline;
+}
+
+
+@media (max-width: 768px) {
+  .main-content {
+      flex-direction: column;
+      padding: 20px;
+  }
+
+
+  .signup-container, .info-container {
+      width: 100%;
+      margin-bottom: 20px;
+  }
+}
+
+
+@media (max-width: 530px) {
+    .hero-overlay h1 {
+        font-size: 1rem; 
+        line-height: 1.2; 
+    }
+
+    .hero-overlay p {
+        font-size: 0.5rem; 
+        line-height: 1.4; 
+    }
+
+    .hero-overlay {
+        padding: 20px; 
+    }
+
+    .logo-image {
+        width: 100px; 
+      }
```

Comment:
CSS: Good job!

Overall is your code well structured. As a reader I would love to have more comments describing the different sections :)
Well done!!!

Reply:
Thank you @gittebe for your comments! These are definitely helpful for future purposes to not just do my own work in my own world but to actually think how someone else perceives/reads my code! So thank you for pointing out the comments, I will definitely take that into consideration. Another point you commented was, why I didn't include the Hero image in my header section - the reasoning was that I wanted to include only my logo in the header section so that I could move the logo non-dependant of the hero image (e.g. using media-queries) + (without using too much  divs and classes) - nothing too strategic, just a preference 😊